_posts/2017/2017-10-06-epages-at-euruko.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "ePages at EuRuKo"
+date: "2017-09-27 09:34:02"
+image: blog-header/budapest-525857_1920.jpg
+image-position: top
+categories: events
+authors: ["Unai"]
+---
+
+Last week our team (Internal tools team) attended the European Ruby Conference in Budapest. Although we use more languages other than Ruby, we love it. It's the base of our team, so it felt natural be there.
+
+## An amazing location
+
+Budapest is awesome. It's a must see city, both during the day and at night. Gorgeous buildings, nice public transport and tasty food.
+
+## First day
+
+We arrived early to the registration, got a nice breakfast and the socialization started. One of the things that I love the most is how nice is this community. Everyone is friendly, everybody wants to share knowledge and listen to others, provide suggestions or just do some small talk.
```

Comment:
We arrived early at the registration, got a nice breakfast and then started networking. One of the things that I loved the most is the nice community. Everyone is friendly, wants to share knowledge, listens to others, provides suggestions and likes doing some small talk.